### PR TITLE
Docs for VaultModule

### PR DESCRIPTION
**Pull request by ajsantander**

### CODE REVIEW

File: packages/synthetix-main/contracts/modules/core/VaultModule.sol [hunk 1]
```diff
@@ -167,7 +169,7 @@ contract VaultModule is IVaultModule {
     }
 
     /**
-     * @dev See {IVaultModule-getPositionDebt}.
+     * @dev Returns the debt for the given account, pool, and collateral type.
```
Comment (dbeal-eth) [inline packages/synthetix-main/contracts/modules/core/VaultModule.sol:172]: should clarify that this function will also trigger a debt distribution chain update, which is why this is a writable function

Comment (dbeal-eth) [reply packages/synthetix-main/contracts/modules/core/VaultModule.sol:172]: possible include reccomendation to use `callStatic` if in ethers (is the function users are looking for)

File: packages/synthetix-main/contracts/modules/core/VaultModule.sol [hunk 2]
```diff
@@ -190,43 +192,56 @@ contract VaultModule is IVaultModule {
     }
 
     /**
-     * @dev See {IVaultModule-getVaultDebt}.
+     * @dev Returns the total debt for the given pool, and collateral type.
```
Comment (dbeal-eth) [inline packages/synthetix-main/contracts/modules/core/VaultModule.sol:195]: same here, function is mutable becuase it triggers a debt distribution change in order to get the debt

**Review by dbeal-eth — APPROVED** [review]

File: packages/synthetix-main/contracts/modules/core/VaultModule.sol (hunk 1 above)
Comment (ajsantander) [reply packages/synthetix-main/contracts/modules/core/VaultModule.sol:172]: Implemented

File: packages/synthetix-main/contracts/modules/core/VaultModule.sol (hunk 2 above)
Comment (ajsantander) [reply packages/synthetix-main/contracts/modules/core/VaultModule.sol:195]: Implemented

**Pull request merged**